networking/multiple-networks/configuring-host-device.adoc

```diff
@@ -12,6 +12,7 @@ namespace into the Pod's network namespace.
 
 include::modules/nw-multus-create-network.adoc[leveloffset=+1]
 include::modules/nw-multus-host-device-object.adoc[leveloffset=+2]
+include::modules/nw-multus-ipam-object.adoc[leveloffset=+2]
```

Comment:
Should add `networking/multiple-networks/configuring-host-device.adoc` to list of assemblies in the `modules/nw-multus-ipam-object.adoc` module now.